examples/simple_experiment.rs

```diff
@@ -13,14 +13,17 @@ fn main() {
     let cpython = GenericScriptingVm::new(&python_bin);
     let pypy = GenericScriptingVm::new(&pypy_bin);
     let luajit = GenericScriptingVm::new(&luajit_bin);
-    let python_bench = Benchmark::new("./benchmarks/binarytrees/binarytrees.py")
-        .tag("abstract_benchmark_name", "binarytrees")
-        .lang_impl(&cpython)
-        .lang_impl(&pypy);
-    let lua_bench = Benchmark::new("./benchmarks/binarytrees/binarytrees.lua")
-        .tag("abstract_benchmark_name", "binarytrees")
-        .lang_impl(&luajit);
-    let exp = expb.benchmark(&python_bench).benchmark(&lua_bench).build();
+    let cpython_bench = Benchmark::new("./benchmarks/binarytrees/binarytrees.py", &cpython)
+        .tag("abstract_benchmark_name", "binarytrees");
```

Comment:
Maybe a teensy nitpick: `abstract_benchmark_name` is quite long. How about something shorter, like `benchmark_name`?